src/product-variants/product-variants.controller.ts

```
@@ -0,0 +1,68 @@
+import { UpdateProductVariantDto } from './dto/requests/update-product-variant.dto';
+
+@Controller('product-variants')
+export class ProductVariantsController {
+  constructor(
+    private readonly productVariantsService: ProductVariantsService,
+  ) {}
+
+  @Post()
+  async create(
+    @Body() createProductVariantDto: CreateProductVariantDto,
+    @LoggedInUser() LoggedInUser: User,
```

Biome (2.3.13)

[error] This parameter is used before its declaration.

The parameter is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)